python/python/tests/utils/utils.py

```diff
@@ -269,3 +275,39 @@ def check_function_list_response(
             break
 
     assert has_lib is True
+
+
+def check_function_stats_response(
+    response: TFunctionStatsResponse,
+    running_function: List[bytes],
+    lib_count: int,
+    function_count: int,
+):
+    """
+    Validate whether `FUNCTION STATS` response contains required info.
+
+    Args:
+        response (TFunctionStatsResponse): The response from server.
+        running_function (List[bytes]): Command line of running function expected. Empty, if nothing expected.
+        lib_count (int): Expected libraries count.
+        function_count (int): Expected functions count.
+    """
+    running_script_info = response.get(b"running_script")
+    if running_script_info == None and len(running_function) != 0:
```

Comment:
Use `is None` instead of `== None`, and `is not None` instead of `!= None` in python